jscomp/others/js_json.ml

```diff
@@ -60,6 +61,31 @@ let test (type a) (x : 'a) (v : a kind) : bool =
   | Array -> Js_array.isArray x 
   | Object -> (Obj.magic x) != Js.null && Js.typeof x = "object" && not (Js_array.isArray x )
 
-
 external parse : string -> t = "JSON.parse" [@@bs.val]
 (* TODO: more docs when parse error happens or stringify non-stringfy value *)
+
+let null : t = Obj.magic Js.null 
+
+let string s : t = Obj.magic s
+
+let number f : t = Obj.magic f
+
+let numberOfInt i : t = Obj.magic i
+
+let boolean b : t = Obj.magic (Js_boolean.to_js_boolean b)
```

Comment:
you can encode ocaml boolean just as int,  no need for coercion, right?
I think for these `to_json` functions, the API should be symmetric, you need provide both, otherwise `encoding` does not make sense?